onnxruntime/python/tools/transformers/notebooks/Inference_GPT2_with_OnnxRuntime_on_CPU.ipynb

```diff
@@ -382,19 +336,25 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 9,
+   "execution_count": 10,
    "metadata": {},
    "outputs": [],
    "source": [
+    "from typing import List, Dict\n",
+    "from onnxruntime import InferenceSession\n",
+    "\n",
+    "from onnxruntime.transformers.io_binding_helper import TypeHelper\n",
+    "from onnxruntime.transformers.io_binding_helper import IOBindingHelper\n",
+    "\n",
     "def inference_with_io_binding(session, config, input_ids, position_ids, attention_mask, past):\n",
     "    output_shapes = Gpt2Helper.get_output_shapes(batch_size=input_ids.size(0),\n",
     "                                                 past_sequence_length=past[0].size(3),\n",
     "                                                 sequence_length=input_ids.size(1),\n",
     "                                                 config=config)\n",
     "    output_buffers = Gpt2Helper.get_output_buffers(output_shapes, device)\n",
     "\n",
-    "    io_binding = Gpt2Helper.prepare_io_binding(session, input_ids, position_ids, attention_mask, past,\n",
-    "                                               output_buffers, output_shapes)\n",
+    "    io_binding = IOBindingHelper.prepare_io_binding(session, input_ids, position_ids, attention_mask, past,\n",
```

Comment:
may be rename the file name, it documents iobinding with GPU usage as well.

Reply:
It's fine to keep the filename until we add an option to run this notebook in GPU. Probably in another pull request.